modules/virt-monitoring-upgrade-status.adoc

```diff
@@ -3,11 +3,10 @@
 // * virt/upgrading-virt.adoc
 
 [id="virt-monitoring-upgrade-status_{context}"]
-= Monitoring upgrade status
+= Monitoring {VirtProductName} upgrade status
 
-The best way to monitor {VirtProductName} upgrade status is to watch the
-cluster service version (CSV) `PHASE`. You can also monitor the CSV conditions
-in the web console or by running the command provided here.
+To monitor {VirtProductName} upgrade status, watch the
+cluster service version (CSV) `PHASE`. You can also monitor the CSV conditions in the web console or by running the command provided here.
```

Comment:
I don't know if it's worth to mention this here or somewhere else in the release note as a know issue but currently we are going to report that the upgrade completed only referring to the upgrade of control plane pods while virt-launcher is upgraded asyncrously.

Reply:
I'm changing the first sentence of this file to `To monitor the status of a {VirtProductName} Operator upgrade...` to hopefully make this clearer. We can add a release note as well if this is a known issue.